Add privacy policy to web site

richardgolding opened the pull request.

richardgolding committed:
WIP privacy statement

richardgolding committed:
Add privacy policy to web site

cabeese commented on client/components/public/Privacy.jsx:
```diff
@@ -0,0 +1,242 @@
+			authority. If you wish to contact us in
+			connection with the exercise of your rights
+			listed above, please email us at&nbsp;
+			<a href="mailto:compliance@greatpuzzlehunt.com">compliance@greatpuzzlehunt.com</a>. We
```
Can you obfuscate the email address? (Is that a thing people still do?) Something like

`<a href='&#109;&#97;ilto&#58;%6&#51;om%7&#48;l&#105;&#97;n%&#54;3%65&#64;gr&#37;65a%74puz%7Al%65%68unt&#46;&#99;&#111;m'>complia&#110;ce&#64;&#103;&#114;&#101;atpuzzlehunt&#46;com</a>`

cabeese commented on client/components/app/Footer.jsx:
```diff
@@ -7,10 +7,13 @@ export default Footer = class Footer extends Component {
             Great Puzzle Hunt &copy; {eventYear}
           </div>
+          <div className="column">
+	    <a href="/privacy" target="_blank">Privacy Policy</a>
```
I feel like this shouldn't open in a new tab. Can you remove `target="_blank"`?

cabeese commented on client/components/public/Privacy.jsx:
```diff
@@ -0,0 +1,242 @@
+			  <li> Establish or exercise our legal rights
+			    or defend against legal claims </li>
+			  <li> Take action regarding fraud, unlawful
+			    or criminal activity </li>n
```
erroneous `n` at end of line

cabeese reviewed (commented):
Looks good - just a couple small things

richardgolding committed:
Fix some review comments

richardgolding commented:
I fixed the typo and removed the target on the privacy policy link.

I'm not inclined to obfuscate this address since we don't obfuscate any of the others on the site, and I've not seen anyone do that for some time anyway

cabeese reviewed (approved):
Great, thanks! I'll trust your judgement on email obfuscation :)
